pkgs/development/libraries/kquickimageedit/default.nix

```diff
@@ -3,35 +3,39 @@
   stdenv,
   fetchFromGitLab,
   extra-cmake-modules,
+  kdePackages,
   qtbase,
   qtdeclarative,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "kquickimageeditor";
-  version = "0.3.0";
+  version = "0.5.1";
 
   src = fetchFromGitLab {
     domain = "invent.kde.org";
     owner = "libraries";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-+BByt07HMb4u6j9bVZqkUPvyRaElKvJ2MjKlPakL87E=";
+    repo = "kquickimageeditor";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-8TJBg42E9lNbLpihjtc5Z/drmmSGQmic8yO45yxSNQ4=";
   };
 
-  nativeBuildInputs = [ extra-cmake-modules ];
+  nativeBuildInputs = [
+    extra-cmake-modules
+  ];
+
   buildInputs = [
+    kdePackages.kirigami
```

Comment:
Looks like this is a Qt6 package, you want to link it to a Qt5 library?  Will qt5compat be enough?

Reply:
`kdePackages.kirigami` is a Qt 6 library; the equivalent for Qt 5 is `libsForQt5.kirigami2`.